documentation/src/test/java/org/hibernate/search/documentation/search/predicate/PredicateDslIT.java

```diff
@@ -746,6 +746,36 @@ public void regexp() {
 		} );
 	}
 
+	@Test
+	public void terms_any() {
+		withinSearchSession( searchSession -> {
+			// tag::terms-any[]
+			List<Book> hits = searchSession.search( Book.class )
+					.where( f -> f.terms().field( "description" )
+							.matchingAny( "robot", "cab" ) )
+					.fetchHits( 20 );
```

Comment:
This predicate will most likely be used on enum fields or embedded id fields.

With that in mind, could you change this example to target the `genre` field? That would be a hint that people don't need to provide values of the field type directly, they can provide values of the property type as well.

I know the other example cannot be changed so easily, so let's leave it as is.